include/exadg/grid/grid.cpp

```diff
@@ -160,10 +160,50 @@ Grid<dim>::create_triangulation(
     {
       // we need to generate the coarse triangulations explicitly
 
-      // TODO
-      // construct all the coarser triangulations
+      // resize the empty coarse triangulations vector
+      coarse_triangulations = std::vector<std::shared_ptr<dealii::Triangulation<dim> const>>(
+        triangulation->n_global_levels());
+
+      unsigned int coarse_level = triangulation->n_global_levels() - 1;
+
+      // lambda function for creating and placing the coarse triangulations
+      auto const lambda_create_coarse_levels = [&](unsigned int              global_refinement,
```

Comment:
The name `lambda_create_coarse_levels` does also not fit very well, since the lambda function creates the triangulation just for one level, not for all. The current name reads like the function creates all coarse levels at once. I suggest `lambda_create_level_triangulation` for consistency with the other names.

Reply:
changed in fe49537a6dd436ad6b49130feb488cd584e5d308.